### PR TITLE
Add variations on the correction scatch->sketch

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -18077,6 +18077,13 @@ scaning->scanning
 scannning->scanning
 scarch->search
 scaricity->scarcity
+scatch->sketch
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Please suggest catch too and equivalents.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -18112,8 +18112,15 @@ scandanavia->Scandinavia
 scaned->scanned
 scaning->scanning
 scannning->scanning
-scarch->search
+scarch->search, scorch,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest scratch too please.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -18112,8 +18112,15 @@ scandanavia->Scandinavia
...
 scaricity->scarcity
+scatch->sketch, catch,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest scratch to all these too, I think first in the list.

I think for this one the suggestions order should be:
scatch->catch, scratch, sketch,

And equivalent for the others.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: e0eee9c

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: e0eee9c

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: e0eee9c

**Pull request merged**